src/modm/platform/extint/sam/module.lb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Erik Henriksson
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+import sys
+
+def init(module):
+    module.name = ":platform:extint"
+    module.description = "External Interrupt"
+
+def prepare(module, options):
+    if options[":target"].identifier.platform != "sam":
+        return False
+
+    module.depends([":cmsis:device", ":platform:clock", ":platform:gpio"])
```

Comment:
```suggestion
    module.depends(":cmsis:device", ":platform:clock", ":platform:gpio")
```
I think this is the CI issue: The docs builder hacks lbuild to grab the modules *before* normalization, therefore the script attempts to concatentate a list with a string.